frontend/src/AppRoutes/index.tsx

```diff
@@ -178,23 +178,25 @@ function App(): JSX.Element {
 	}, [pathname]);
 
 	useEffect(() => {
-		try {
-			const isThemeAnalyticsSent = getLocalStorageApi(
-				LOCALSTORAGE.THEME_ANALYTICS,
-			);
-			if (!isThemeAnalyticsSent) {
-				trackEvent('Theme Analytics', {
-					theme: isDarkMode ? THEME_MODE.DARK : THEME_MODE.LIGHT,
-					user: pick(user, ['email', 'userId', 'name']),
-					org,
-				});
-				setLocalStorageApi(LOCALSTORAGE.THEME_ANALYTICS, 'true');
+		if (user && user?.email && user?.userId && user?.name) {
+			try {
+				const isThemeAnalyticsSent = getLocalStorageApi(
+					LOCALSTORAGE.THEME_ANALYTICS_V1,
+				);
+				if (!isThemeAnalyticsSent) {
+					trackEvent('Theme Analytics', {
+						theme: isDarkMode ? THEME_MODE.DARK : THEME_MODE.LIGHT,
```

Comment:
We wanted to move away from trackEvent and use logEvent 

cc: @makeavish

Reply:
This is a NIT for now, approved the PR

Reply:
we will anyways remove this event in a while so should be okay for now